pkg/ocicni/util_freebsd.go

```diff
@@ -0,0 +1,89 @@
+//go:build freebsd
+// +build freebsd
+
+package ocicni
+
+import (
+	"fmt"
+	"net"
+	"os/exec"
+	"strings"
+)
+
+var defaultNamespaceEnterCommandName = "jexec"
+
+type nsManager struct {
+	nsenterPath string
```

Comment:
since we have a separate one for freebsd, shouldn't this be jenterPath? that goes for most of the references to `ns` in this file

Reply:
Good point. I renamed things in this file to clarify (hopefully)